backend/qemu.pm

```diff
@@ -762,6 +765,7 @@ sub start_qemu {
         $self->{proc}->configure_blockdevs($bootfrom, $basedir, $vars);
         $self->{proc}->configure_pflash($vars);
     }
+    bmwqemu::diag('initializing block device images');
```

Comment:
You could start the message with a capital letter, just for being consistent with the previous message.

Reply:
done